variables.tf

```diff
@@ -3,11 +3,47 @@
 ##############################################################################
 
 variable "name" {
-  description = "The name to give the newly provisioned VPC. Only used if 'create_vpc' is true."
+  description = "The name to give the provisioned VPC. Only used if 'create_vpc' is true."
```

Comment:
hmm, name is also used for other resources, not just the VPC. I would update this to:

"The string to use for the naming of VPC resources."

I think you should also then add a new variable called `vpc_name` which can be used to explicitly name the VPC.

Reply:
This module is being consumed in the SLZ module. Any changes which we make here with `name` will impact that too. I don't think so that's needed. Hence my proposal is I can rename the description to the following:

```
The string to use for the naming of VPC and its resources.
```